hs_core/views/utils.py

```
@@ -13,7 +19,10 @@ def authorize(request, res_id, edit=False, view=False, full=False, superuser=Fal
     the parameter list, then this returns True else False.
     """
     user = get_user(request)
-    res = hydroshare.utils.get_resource_by_shortkey(res_id)
+    try:
+        res = hydroshare.utils.get_resource_by_shortkey(res_id)
+    except:
```

another bare except that shouldn't be here